### DISCUSSION TITLE
Vercel phone number limitations

### DISCUSSION THREAD

**Discussion by OhadBaehr** [vercel/community#2701] [Help] [answered]
You can't have multiple accounts under the same phone number on Vercel.

I have apps launched on Vercel from a different GitHub account.

I have my current project which I need to launch using my current account.

Vercel basically limits me to either
1) Delete the apps currently on Vercel, delete the other account and redeploy from this account
2) Change my phone number
3) Be forced to use my old account

None of those options make any sense.

Please remove the phone number limit.

**Comment by amyegan** [comment, marked as answer]
Hi @OhadBaehr! Please send an email to registration@vercel.com with your email address, phone number, and summary of why you need a second account with the same phone number. They will work with you to get everything set up properly!

**Comment by liznelson** [comment]
This seems to be a problem with individuals working with both an enterprise and a personal hobby account. Any plans to address this?
